lib/src/Uncertainty/Distribution/Wishart.cxx

```diff
@@ -266,7 +266,7 @@ Wishart::PointWithDescriptionCollection Wishart::getParametersCollection() const
   point[index] = nu_;
   description[index] = "nu";
   point.setDescription(description);
-  PointWithDescriptionCollection parameters(point.getDimension());
+  PointWithDescriptionCollection parameters(1, point);
   return parameters;
 }
 
```

Comment:
This method is COMPLETELY WRONG, and I am the only responsible for that! The Wishart distribution is a multidimensional distribution, and as such, the getParametersCollection() method should return a collection of size dim+1. The difficult part is to describe the marginal parameters for such a distribution, as its natural range is over SPD matrices and not vectors. IMO a NotYetImplementedException should be thrown here, or at least a collection of a correct size should be returned. Or you are ready to wait a little bit for a contribution from my side (check your email! ;-))

Reply:
I removed my wishart fix waiting for a better alternative.